### DISCUSSION TITLE
Test Ad Server URL requested

### DISCUSSION THREAD

**Discussion by rrirower** [rrirower/DP2SceneGraphWizard#34] [General]
I'm in the process of testing Ads.  I've been using the test URL provided by Roku.  If you have an Ad server URL I can test with, please respond to this post.

Thank you...

**Comment by 37media** [comment]
Hey, I have an ad server if you need access to a URL

**Reply by rrirower** [reply]
Thanks.  If you can, please post it here, or email to rrirower@charter.net

**Comment by 37media** [comment]
Did u still need an ad server?   Sorry I would extend it monthly but
stopped after the conversion.

**Reply by rrirower** [reply]
No.  Thanks.  I'm good for now.
